packages/demo-next/package.json

```diff
@@ -7,8 +7,7 @@
   "author": "Kendall Strautman",
   "scripts": {
     "dev-next": "next",
-    "dev": "node server.js",
-    "develop": "npm run dev",
+    "develop": "node server.js",
```

Comment:
aw I miss the dev command lol

Reply:
lol we removed it because `lerna run dev` would start the server